packages/stark-ui/src/modules/collapsible/components/collapsible.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, EventEmitter, HostBinding, Inject, Input, OnInit, Output } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-collapsible";
+
+/**
+ * Component to display an accordion around included html using angular material's expansion panel.
+ */
+@Component({
+	selector: "stark-collapsible",
+	templateUrl: "./collapsible.component.html"
+})
+export class StarkCollapsibleComponent implements OnInit {
+	/**
+	 * Adds class="stark-app-logo" attribute on the host component
+	 */
+	@HostBinding("class")
+	public class: string = componentName;
+
+	/**
+	 * ID to set to the collapsible
+	 */
+	@Input()
+	public htmlId: string;
+
+	/**
+	 * Icon to use in the header (based on Material Design Icons)
+	 */
+	@Input()
+	public icon: "chevron-right" | string = "chevron-right";
+
+	/**
+	 * Boolean value triggering the collapsing of the collapsible
+	 */
+	@Input()
+	public isExpanded: boolean = false;
+
+	/**
+	 * Output value giving the collapse state of the collapsible
+	 */
+	@Output()
+	public isExpandedChange: EventEmitter<boolean> = new EventEmitter();
```

Comment:
Could you please rename this output to `expanded` in order to follow the [Angular Style Guide](https://angular.io/guide/styleguide#dont-prefix-output-properties).

And can you also show such feature in the demo page? You can simply add a label next to the "Toggle Collapsible" button where the `expanded` value is displayed and it changes everytime the collapsible expand/collapses. What do you think?